### PR TITLE
FIX coverage by using jest projects over multiple npm scripts

### PR DESCRIPTION
**Pull request by ndelangen**
Issue: -coverage was broken-

## What I did
REMOVE the multitude of npm tests scripts
ADD jest projects to run a selection with a different config

## How to test
run `npm test -- --coverage`

### CODE REVIEW

**Comment by tmeasday** [comment]
Oh wow, that's awesome great work!

**Review by tmeasday — APPROVED** [review]

File: examples/react-native-vanilla/storybook/stories/index.js [hunk 1]
```diff
@@ -13,13 +13,13 @@ storiesOf('Welcome', module).add('to Storybook', () => <Welcome showApp={linkTo(
...
   .addDecorator(getStory => <CenterView>{getStory()}</CenterView>)
-  .add('with text', () => (
+  .add('with text', () =>
     <Button onPress={action('clicked-text')}>
       <Text>Hello Button</Text>
     </Button>
-  ))
-  .add('with some emoji', () => (
+  )
+  .add('with some emoji', () =>
```
Comment (shilman) [inline examples/react-native-vanilla/storybook/stories/index.js]: @ndelangen if our `getstorybook` templates are failing our linting rules, we should probably fix our templates or fix our linting rules. Actually I prefer the original style... ;-)

File: package.json [hunk 2]
```diff
@@ -23,11 +23,7 @@
     "lint:js": "eslint . --cache --cache-location=.cache/eslint --ext .js,.jsx,.json",
     "lint:md": "remark .",
     "publish": "lerna publish",
-    "test": "npm run test:libs && npm run test:apps",
-    "test:apps": "npm run test:react-native-vanilla",
-    "test:libs": "jest",
-    "test:watch": "npm run test:libs -- --watch",
-    "test:react-native-vanilla": "cd examples/react-native-vanilla && jest"
+    "test": "jest --projects ./ ./examples/react-native-vanilla"
```
Comment (shilman) [inline package.json]: @ndelangen is this redundant with L17 of `jest.config.js` above?

File: examples/react-native-vanilla/storybook/stories/index.js (hunk 1 above)
Comment (ndelangen) [inline examples/react-native-vanilla/storybook/stories/index.js]: True, we should fix em. I'm pretty sure this is not a configurable option of prettier, but I may be wrong. Feel free to fix and experiment if this is configurable.

File: package.json (hunk 2 above)
Comment (ndelangen) [inline package.json]: Unfortunately no, there's a known bug in jest projects that make it so it needs to be configured via CLI 😢

File: examples/react-native-vanilla/storybook/stories/index.js (hunk 1 above)
Comment (shilman) [inline examples/react-native-vanilla/storybook/stories/index.js]: @ndelangen turns out you already fixed them with your big linting merge. the problem is that the fix hasn't been released yet

Comment (ndelangen) [inline examples/react-native-vanilla/storybook/stories/index.js]: We can release, if we move some stuff over to 3.0.3:
https://github.com/storybooks/storybook/milestone/6

Comment (tmeasday) [inline examples/react-native-vanilla/storybook/stories/index.js]: I think the idea of prettier is you don't configure it, you just enjoy not quibbling over this stuff.

**Pull request merged**